protostar/commands/call_command.py

```diff
@@ -1,24 +1,44 @@
 import json
+from dataclasses import dataclass
 from logging import Logger
-from typing import Any, Optional, Dict
+from typing import Any, Optional
 
 from starknet_py.net.gateway_client import GatewayClient
 
 from protostar.cli import (
     NetworkCommandUtil,
+    OutputCommandUtil,
     ProtostarArgument,
     ProtostarCommand,
 )
+from protostar.io import LogColorProvider, StructuredMessage
 from protostar.starknet_gateway import GatewayFacadeFactory
 
 
+@dataclass
+class SuccessfulCallMessage(StructuredMessage):
+    response: Any
+
+    def format_human(self, fmt: LogColorProvider) -> str:
```

Comment:
```suggestion
    def format_human(self, fmt: LogColorProvider) -> str:
```
`get_console_output` maybe?

Reply:
get_console_output and as_dict are not consistently named.

I thought about `display_human` and `display_object` like Rust's `Display` trait, but I think format looks better.